src/utils/checkDisplayType.ts

```diff
@@ -2,16 +2,13 @@
 // put a check for the above things later
 
 export function checkDisplayType({
-  msgToBeSent,
   usersWithMatchingRole,
 }: {
   msgToBeSent?: string;
   usersWithMatchingRole: string[];
 }) {
   if (usersWithMatchingRole.length > 0) {
-    const returnString = msgToBeSent ? msgToBeSent : "";
-
-    return `${returnString} ${usersWithMatchingRole} \n \`Disclaimer: Very soon all the users will be tagged individually in a separate(new) message!\``;
+    return `${usersWithMatchingRole}`;
```

Comment:
why did we remove the returnString? mention each also includes the message that the users send, if we just return the users with matching roles then we would never get the message.

Reply:
resolved it in last commit